src/main/java/teammates/ui/pagedata/InstructorFeedbackSessionsPageData.java

```diff
@@ -222,21 +222,32 @@ public FeedbackSessionsCopyFromModal getCopyFromModal() {
     }
 
     /**
-     * Creates a list of options (STANDARD and TEAMEVALUATION). If defaultSessionType is null,
-     *     TEAMEVALUATION is selected by default.
-     * @param defaultSessionType  either STANDARD or TEAMEVALUATION, the option that is selected on page load
+     * Creates a list of options (STANDARD, TEAMEVALUATION, and OPTIMIZEDTEAMEVALUATION). If defaultSessionType is null,
+     *     OPTIMIZEDTEAMEVALUATION is selected by default.
+     * @param defaultSessionType  either STANDARD, TEAMEVALUATION or OPTIMIZEDTEAMEVALUATION,
+     *                            the option that is selected on page load
      */
     private List<ElementTag> getFeedbackSessionTypeOptions(String defaultSessionType) {
```

Comment:
ummMMMmm is it possible for u to change the name of this function? 
and also you might want to specify in the comment of this function that this function creates a list of options (...) for the question template of the session.

Reply:
The reason why I did not is because this is being changed by #8518 , and I want to avoid conflicts with that PR.

Reply:
uhhh i see